pages/index.js

```diff
@@ -1,20 +1,20 @@
 import useSWR from "swr";
-
+import ArtPieces from "@/components/ArtPieces";
 const fetcher = (url) => fetch(URL).then((response) => response.json());
```

Comment:
You could implement an appropriate error handling, like it is shown in the [SWR Docs](https://swr.vercel.app/docs/error-handling). Maybe you think about it for a future feature...